typescript/src/gapic-generator-typescript.ts

```diff
@@ -84,7 +84,27 @@ yargs.usage('Usage: $0 -I /path/to/googleapis');
 yargs.usage('  --output_dir /path/to/output_directory');
 yargs.usage('  google/example/api/v1/api.proto');
 
-const argv = yargs.argv;
+export interface IArguments {
+  [x: string]: unknown;
+  outputDir?: string;
+  grpcServiceConfig?: string | undefined;
```

Comment:
I think when you use `?` it already means "can be undefined" so should be enough to say just `grpcServiceConfig?: string` without `| undefined`.  Same for others.

Reply:
SG. remove `undefined`.